.github/workflows/platformDeploy.yml

```diff
@@ -387,3 +387,32 @@ jobs:
           DESKTOP: ${{ needs.desktop.result }}
           IOS: ${{ needs.iOS.result }}
           WEB: ${{ needs.web.result }}
+
+  deployChecklist:
```

Comment:
One side-effect of this will be that the new checklist won't show up until after deploys finish. I guess that's not really ideal...

Reply:
Chatted with @luacmartins and @Julesssss and they agreed this is an improvement on what we have